cmd/manager/main.go

```diff
@@ -136,6 +138,8 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx := context.TODO()
```

Comment:
Why TODO instead of Background()?

This should be wired to Ctrl+C so that it tries to gracefully finish the program right?
Not for this PR, but if we are to leave a TODO maybe we could leave a comment with the intended follow up:
https://medium.com/@matryer/make-ctrl-c-cancel-the-context-context-bd006a8ad6ff

BTW I think we should really move these code into `pkg/` so that the manager is ONLY ever configured once, not 4 times (here and then again for e2e tests and twice for int tests)

@s-urbaniak could we get that prioritised soon? ^ I do not think is so big it needs a quaterly planned thing.
1-2 days should be more than enough

Reply:
@helderjs @josvazg i believe we don't need to priorize here anything the code is there a few lines below. `ctrl.SetupSignalHandler()` already wires things to `SIGINT` and `SIGTERM`

Reply:
fixed in https://github.com/mongodb/mongodb-atlas-kubernetes/pull/1529/commits/72c02478d062f3d9a17db77dad582d601a1a1b5c which I pushed to this PR